admin-dev/themes/new-theme/scss/components/_material_choice_tree.scss

```diff
@@ -98,3 +98,9 @@
     }
   }
 }
+
+.category-dropdown-button {
```

Comment:
How about `category-choice-tree-button` to match naming with `CategoryChoiceTreeButtonType`?